definitions/ext-ddmysql/golden_metrics.yml

```diff
@@ -0,0 +1,1462 @@
+bufferPoolDirty:
+  title: Buffer Pool Dirty (byte)
+  displayAsValue: false
+  query:
+    select: mysql.innodb.buffer_pool_dirty
+    from: Metric
```

Comment:
Golden Signals has no support for shimming metrics yet.
So there are two possibilities here:
- keep these queries and don't have golden signals (lookout view) until they implement support for it.
- don't use the shimming and use the underlying queries and have golden signals

Reply:
What are "shimming metrics" if I may ask?

Reply:
Shimmings is something NRDB team is working on. But basically the objective is to always use `FROM Metric` in the queries.
So it converts "old" ones like `FROM AwsEc2Sample` into a query that works using `FORM Metric`. This is invisible for users but sadly we have a service that "knows" about this things. And hence we should too :(

That said that service should implement this logic and we should not really care about it.